dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -13,13 +13,15 @@ const CONSTANTS = require('../../../../constants');
 const findAllAmendmentsByStatus = async (status) => {
   try {
     const collection = await db.getCollection('tfm-facilities');
-    const amendment = await collection.aggregate([
-      { $project: { _id: 0, amendments: '$amendments' } },
-      { $unwind: '$amendments' },
-      { $match: { 'amendments.status': status } },
-      { $group: { _id: '$_id', amendments: { $push: '$amendments' } } },
-      { $project: { _id: 0, amendments: 1 } },
-    ]).toArray();
+    const amendment = await collection
```

Comment:
These changes are just from running prettier formatter on the file that I had already expected to be done with the prettierrc.json file in the codebase